client/src/components/CreateCampaignStep1.js

```diff
@@ -20,49 +29,49 @@ const CreateCampaignStep1 = props => {
     });
     router.push('/new-campaign/optional-info');
   };
-
   return (
-    <div className="add_address_wrapper">
-      {formattedAddress && (
-        <form onSubmit={setAddressAndName}>
-          <div className="form-group">
-            <label className="required">Address</label>
-            <input
-              type="text"
-              className="form-control"
-              value={formattedAddress}
-              name="address"
-              readOnly
-            />
+    <Row>
+      <Col xs={12}>
+        {formattedAddress && (
+          <form onSubmit={setAddressAndName} className="create-campaign-form">
+            <PageHeader>Create Campaign</PageHeader>
+            <FormGroup controlId="addressAndName">
+              <ControlLabel>Address</ControlLabel>
+              <FormControl type="text" value={formattedAddress} name="address" readOnly />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel className="create-campaign-control-label">Campaign Name</ControlLabel>
+              <div className="requiredtool" />
+              <FormControl type="text" name="campaignName" required />
+            </FormGroup>
+            <br />
+            <Button bsStyle="remove-default" className="next-button fr" type="submit">
+              Next
+            </Button>
+          </form>
+        )}
+        {error && (
+          <div className="text-center">
+            <p>
+              {
+                'There was an issue with the address you provided. It may be associated with a preexisting campaign.'
+              }
+            </p>
+            <br />
           </div>
-          <div className="form-group">
-            <label className="required">Campaign Name</label>
-            <div className="requiredtool" />
-            <input type="text" className="form-control campaign" name="campaignName" required />
+        )}
+        {!formattedAddress && (
+          <div className="text-center">
+            <p>
+              {'Add an address to start.'}{' '}
```

Comment:
combine this trailing space ![bitmoji](https://render.bitstrips.com/v2/cpanel/42615a58-226a-495f-b69f-68517b5a87e2-9e17bbb5-ab70-4793-bdf9-b6ef51f2a3a3-v1.png?transparent=1&palette=1&width=246)

Reply:
ok